CITATION.cff

```diff
@@ -0,0 +1,52 @@
+# doc: https://citation-file-format.github.io/
+# schema: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+cff-version: 1.2.0
+
+title: "neurobagel cli"
+
+version: 0.2.0
```

Comment:
```suggestion
version: 0.1.0
```

Reply:
hehe, aha. daisy chainging mistakes is no good